python/ray/autoscaler/_private/prom_metrics.py

```diff
@@ -202,6 +202,22 @@ def __init__(self, registry: Optional[CollectorRegistry] = None):
                 namespace="autoscaler",
                 registry=self.registry,
             )
+            self.cluster_resources: Counter = Gauge(
+                "cluster_resources",
+                "Total logical resources in the cluster.",
```

Comment:
Does this total include pending? Otherwise we may not need it (we already track this from core)

Reply:
added a comment, it's slightly different from an eventual consistency perspective.